### PR TITLE
Update SHOW STATUS docs

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

https://github.com/pingcap/docs/pull/15363 for release-7.5

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qiancai — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

File: sql-statements/sql-statement-show-status.md [hunk 1]
```diff
@@ -5,56 +5,66 @@ summary: An overview of the usage of SHOW [GLOBAL|SESSION] STATUS for the TiDB d
...
+| Ssl_verify_mode               | 0                                    |
+| Ssl_version                   |                                      |
+| Uptime                        | 1409                                 |
+| ddl_schema_version            | 116                                  |
+| last_plan_binding_update_time | 0000-00-00 00:00:00                  |
+| server_id                     | 61160e73-ab80-40ff-8f33-27d55d475fd1 |
++-------------------------------+--------------------------------------+
+10 rows in set (0.00 sec)
+
+sql> SHOW GLOBAL STATUS;
```
Comment (yahonda) [inline sql-statements/sql-statement-show-status.md:46]:
```suggestion
mysql> SHOW GLOBAL STATUS;
```

Comment (yahonda) [reply sql-statements/sql-statement-show-status.md:46]: I have overlooked this when I reviewed https://github.com/pingcap/docs/pull/15363 .

**Review by yahonda — COMMENTED** [review]
Found a minor typo about the command prompt. Others are fine.

**Review by yahonda — APPROVED** [review]
Looks good.

**Pull request merged**